source/type-validation.txt

```diff
@@ -58,15 +58,43 @@ an equality comparison with an ``Integer`` value ``1324324``:
    }
 
 Because ``value`` is an ``Integer`` but ``trip_status`` is a ``String``
-in the MongoDB documents, the plugin shows a warning about the invalid
-type:
+in the MongoDB documents, the side panel shows a warning about the invalid
+type under :guilabel:`Correctness Warnings`:
 
-``"Integer" (type of provided value) can not be assigned to "String"
-(type of "trip_status")``
+.. image:: /images/type-validation.png
+   :alt: Type Validation Warning
+   :width: 400px
 
 To resolve the invalid type, change the value to a ``String``. For
 example: ``"start"``.
 
+Disable Type Validation Warnings 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
(info typing) 
Could move to a task page. Happy to chat if you want.

Reply:
Moved to its own task page!